spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewritePositionDeleteFilesAction.java

```diff
@@ -380,7 +380,7 @@ public void testRemoveDanglingDeletes() throws Exception {
     List<Object[]> actualRecords = records(table);
     List<Object[]> actualDeletes = deleteRecords(table);
     assertEquals("Rows must match", expectedRecords, actualRecords);
-    assertThat(actualDeletes).as("Should be no new position deletes").hasSize(0);
+    assertThat(actualDeletes).as("No new position deletes").isEmpty();
```

Comment:
Do we need `no`?  I wonder what the assert message is, maybe it will already say 'expected new position deletes to be empty'

Reply:
Remove. I tried to mock a failure, by changing isEmpty to isNotEmpty. Here is the messages with `No` started:

```
TestRewritePositionDeleteFilesAction > testRemoveDanglingDeletes() > catalogName = testhive, implementation = org.apache.iceberg.spark.SparkCatalog, config = {type=hive, default-namespace=default, cache-enabled=false}, fileFormat = PARQUET FAILED
    java.lang.AssertionError: [No new position deletes] 
    Expecting actual not to be empty
        at org.apache.iceberg.spark.actions.TestRewritePositionDeleteFilesAction.testRemoveDanglingDeletes(TestRewritePositionDeleteFilesAction.java:383)
```